### PR TITLE
Corrections and updates to DROTAG alternative

### PR DESCRIPTION
**Pull request by ptsneves**

### CODE REVIEW

File: common/source/docs/common-geotagging-images-with-mission-planner.rst [hunk 1]
```diff
@@ -308,37 +309,48 @@ working with specific cameras:
 Alternative Commercial Hardware Solutions
 =========================================
 
-DROTAG - Airborne Projects
---------------------------
+DROTAGx - In-Camera Card Photo Tagging - Airborne Projects
+---------------------------
+.. note::
+
+   This section is in no way endorsed or verfied by the Ardupilot Wiki or it's
```
Comment (lvale) [inline common/source/docs/common-geotagging-images-with-mission-planner.rst]: verified

Comment (lvale) [inline common/source/docs/common-geotagging-images-with-mission-planner.rst]: Well. I've tested :)

File: common/source/docs/common-geotagging-images-with-mission-planner.rst [hunk 2]
```diff
@@ -308,37 +309,48 @@ working with specific cameras:
...
 other tips.
 
-.. image:: https://www.airborneprojects.com/wp-content/uploads/2016/06/text_image_1-1200x548.jpg
-    :target: https://www.airborneprojects.com/product/drotag-onboard-image-tagging-computer/
-    :width: 100%
-    
-.. image:: https://www.airborneprojects.com/wp-content/uploads/2016/06/3.jpg
-    :target: https://www.airborneprojects.com/product/drotag-onboard-image-tagging-computer/
-    :width: 50%
+DROTAGx is used by several nationional government land management organizations as well as
```
Comment (lvale) [inline common/source/docs/common-geotagging-images-with-mission-planner.rst]: national

File: common/source/docs/common-geotagging-images-with-mission-planner.rst [hunk 3]
```diff
@@ -308,37 +309,48 @@ working with specific cameras:
...
+by the United States armed forces. Many profession surveying companies also
```
Comment (lvale) [inline common/source/docs/common-geotagging-images-with-mission-planner.rst]: professional

**Comment by ptsneves** [comment]
@lvale I had some difficulties trying to render the page and ended up manually copy pasting to the github editor and de-synchronizing the text. Shame on me.

**Comment by rmackay9** [comment]
Merged, hope it's all OK.

**Pull request merged**